packages/p2p/README.md

```diff
@@ -2,84 +2,57 @@
   @deriv/p2p
 </h1>
 
-**In this document**
+This repository is a workspace of [deriv-app](../../README.md) monorepo and covers the components and logics of [p2p](https://app.deriv.com/cashier/p2p) from cashier menu in app.deriv.com .
 
--   [Pre-installation](#pre-installation)
--   [Editor helpers](#editor-helpers)
--   [Quick start](#quick-start)
+> to access the p2p section you need to login and switch to your real account.
 
-## Pre-installation
+**these are the routes used in app.deriv.com for this package:**
 
--   node
--   npm
-
-## Editor helpers
-
--   Prettier setup in your editor https://prettier.io/
--   Stylelint setup in your editor https://stylelint.io/
--   Eslint setup in your editor https://eslint.org/
-
-## Quick start
-
-1.  **Install your dependencies:**
-
-    ```sh
-    npm ci
-    ```
-
-2.  **To build publish file:**
+```
+    cashier_p2p: '/cashier/p2p',
+    cashier_p2p_verification: '/cashier/p2p/verification',
+    cashier_pa_transfer: '/cashier/payment-agent-transfer',
```

Comment:
We don't use the payment agent route, this shouldn't be here.

Reply:
thanks.I'll remove it